mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerStressTest.java

```diff
@@ -103,7 +99,8 @@ public void event(Event event) {
 
                     mailboxManager.startProcessingRequest(mailboxSession);
                     MessageManager m = mailboxManager.getMailbox(path, mailboxSession);
-                    ComposedMessageId messageId = m.appendMessage(new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes()), new Date(), mailboxSession, false, new Flags());
+                    ComposedMessageId messageId = m.appendMessage(MessageManager.AppendCommand.builder()
```

Comment:
in this particular case, we forged a mail as a string because we don't have a good builder to build a mail object then pass it to appendMessage. I would be much more interesting to provide such a builder.
```
AppendCommand.builder().subject("test").content("testmail")
```
WDYT ?

Reply:
Imo it should not be a responsability of this object... 

We already have too much mail builders... Mime4j, javax, etc... I would value composing our new AppendCommand with such tools, thus achieving the result you want by composition, and with minimal code changes.

Also I will do it in a separated PR: I may need some mime4j reworking...

Reply:
Actually, I started this task yesterday and accepting a MimeMessage allows to use MimeMessageBuilder, it's more than enough IMO.